components/ucd/category/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "unic-ucd-category"
+version = "0.0.1"
+authors = ["The UNIC Project Developers"]
+homepage = "https://github.com/behnam/rust-unic/"
+repository = "https://github.com/behnam/rust-unic/"
+license = "MIT/Apache-2.0"
+keywords = ["text", "unicode"]
+description = "UNIC - Unicode Character Database"
+readme = "README.md"
```

Comment:
Drop this if we don't have a `README.md` on this crate. (The small crates don't have one yet.)